lib/utils/ignore.ts

```diff
@@ -120,7 +120,7 @@ export class FileIgnorer {
 		});
 
 		return !_.some(ignoreTypes, ({ handle }) => handle.ignores(relFile));
-	};
+	}; // tslint:disable-line:semicolon
```

Comment:
Here there was a fight between prettier and resin-lint (tslint). If I deleted the semicolon to appease resin-lint, prettier would get cross and automatically re-add it.

Reply:
That is indeed weird. I'm trying to get the new VS Code tslint plugin working for past 2 hours, but I'm thinking whether it could also be related to TS version and/or node version. Will update when I find out more

Reply:
@JackuB, I was able to get the VS Code tslint extension to work by adding a `tslint.json` file to the project root, with these contents:
```
{
	"extends": "./node_modules/resin-lint/config/tslint.json",
	"rules": {
		"ignoreDefinitionFiles": false
	}
}
```
It seems to be ignoring `.d.ts` files, even after I added the `"ignoreDefinitionFiles": false` setting above. But it's working for the regular `.ts` files.

Reply:
I had some other VS Code glitch, because the extension had issues even with the node-resin-lint repo, where it worked before.

But, I was able to simulate some different results with tslint depending on the version of typescript (not just this one, found even some `empty block` and others warnings). Working on a fix in resin-lint to isolate it. Anyway, removing the `// tslint:disable-line:semicolon` with typescript@3.3.3333 is not resulting in error anymore. But it's sad that tslint rules are influenced by typescript version so wildly.

Reply:
Loved the `"extends": "./node_modules/resin-lint/config/tslint.json",` part @pdcastro :+1: 
I will start using it everywhere I'm using resin-lint to also get the lining errors beforehand in SublimeText :)

Reply:
Sounds right, I've edited my sample `tslint.json` above to extend `tslint-prettier.json`. The `.d.ts` files still seem to be ignored, though.